### DISCUSSION TITLE
The source code for version 1.7.10 of the mod

### DISCUSSION THREAD

**Discussion by sdddddf80** [MrTJP/ProjectRed#1810] [Announcements]
Hello. I wonder where could I get the source code for version 1.7.10 of the mod as I don't find a 1.7.10 branch in this repositories?

**Comment by simon816** [comment]
Using git blame I went back to the last stable 1.7.10 release, which appears to be version `4.6.2`

tree: https://github.com/MrTJP/ProjectRed/tree/887c72011308cad76a7a4de40a92a666e0b107dc

**Comment by sdddddf80** [comment]
> Using git blame I went back to the last stable 1.7.10 release, which appears to be version `4.6.2`
> 
> tree: https://github.com/MrTJP/ProjectRed/tree/887c72011308cad76a7a4de40a92a666e0b107dc

Thank you very much! But from the perspective of curseforge, the final version under 1.7.10 is 4.7.0pre12.95. Is there a significant difference between the them?

**Comment by simon816** [comment]
Ah right, I skipped over those `pre` versions

This commit appears to be the last for 1.7.10 https://github.com/MrTJP/ProjectRed/commit/7616a956d0ac48e1ec8db96f992e7cbf010ec442 (tree: https://github.com/MrTJP/ProjectRed/tree/7616a956d0ac48e1ec8db96f992e7cbf010ec442)

**Comment by sdddddf80** [comment]
@simon816 I'm really sorry to trouble you again, but I would like to ask, whether I use nbtedit to modify the blueprint's nbt or obtain `IntegratedCircuit` through tileEntity, and then obtain a specific `CircuitPart` through `getPart (x, y)` and modify it, the entire circuit will not refresh. 
![image](https://github.com/MrTJP/ProjectRed/assets/53573250/ee80b88d-ac01-4278-8c28-7ddb8ea4f8cb)
Do you know anything about this?
